src/main/java/io/vertx/redis/impl/AbstractRedisClient.java

```diff
@@ -115,7 +113,8 @@ final void sendJsonObject(final String command, final JsonArray args, final Hand
       case "psubscribe":
         // in this case we need also to register handlers
         if (redisArgs == null) {
-          resultHandler.handle(new RedisAsyncResult<>("at least one pattern is required!"));
+          vertx.getOrCreateContext().runOnContext(v ->
```

Comment:
Instead of calling vertx.getOrCreateContext() multiple times in the same method, it would be nicer to call it once at the top and just use the context obtained later on.